src/metabase/pulse/render/body.clj

```diff
@@ -226,15 +226,16 @@
    timezone-id :- [:maybe :string]
    card
    _dashcard
-   {:keys [rows viz-settings format-rows?] :as data}]
-  (let [[ordered-cols ordered-rows] (order-data data viz-settings)
-        data                        (-> data
+   {:keys [rows viz-settings format-rows?] :as unordered-data}]
+  (let [[ordered-cols ordered-rows] (order-data unordered-data viz-settings)
+        data                        (-> unordered-data
                                         (assoc :rows ordered-rows)
                                         (assoc :cols ordered-cols))
         table-body                  [:div
                                      (table/render-table
-                                      (color/make-color-selector data viz-settings)
-                                      (common/column-titles ordered-cols (::mb.viz/column-settings viz-settings) format-rows?)
+                                      (color/make-color-selector unordered-data viz-settings)
```

Comment:
pass the unordered data here so that the color selector contains necessary data for every column, even ones hidden by viz-settings.

Reply:
This fixes conditional formatting not showing up on rows when the condition is on a hidden column.